e2e_tests/tests/cluster/test_users.py

```diff
@@ -743,21 +733,22 @@ def test_tensorboard_creation_and_listing(clean_auth: None, login_admin: None) -
 def test_command_creation_and_listing(clean_auth: None, login_admin: None) -> None:
     creds1 = create_test_user(True)
     creds2 = create_test_user(True)
+    session1 = api_utils.determined_test_session(credentials=creds1)
+    session2 = api_utils.determined_test_session(credentials=creds2)
 
-    with logged_in_user(creds1):
-        command_id1 = run_command()
+    command_id1 = run_command(session=session1)
 
-    with logged_in_user(creds2):
-        command_id2 = run_command()
+    command_id2 = run_command(session=session2)
 
-    with logged_in_user(creds1):
-        output = extract_columns(det_run(["cmd", "list"]), [0, 1])
-        assert (command_id1, creds1.username) in output
-        assert (command_id2, creds2.username) not in output
+    cmds = bindings.get_GetCommands(session1, users=[creds1.username]).commands
+    output = [(cmd.id, cmd.username) for cmd in cmds]
```

Comment:
these are not exactly the same this is also testing what the cli argument does but that i think is not the goal here